backend/data/src/main/kotlin/io/tolgee/email/EmailService.kt

```diff
@@ -63,7 +64,7 @@ class EmailService(
     context.setVariable(ThymeleafEvaluationContext.THYMELEAF_EVALUATION_CONTEXT_CONTEXT_VARIABLE_NAME, tec)
 
     val html = templateEngine.process(template, context)
-    val subject = extractEmailTitle(html)
+    val subject = subject ?: extractEmailTitle(html)
```

Comment:
Note: Setting a subject would break internationalization of emails. I guess this is needed for the backcompat layer, but... not ideal as it makes the service contract loose here :(

I don't think the template allows the subject being a variable directly but that could be mitigated with a passthrough i18n string; i.e. `email.legacy.subject-passthrough = "{subject}"`